myproject/views.py

```diff
@@ -1,35 +1,44 @@
 from pyramid.view import view_config
-from pyramid.response import Response
-
-
-# @view_config(route_name='home', renderer='templates/mytemplate.pt')
-# def my_view(request):
-#     return {'project': 'MyProject'}
-
 import os
 
 HERE = os.path.dirname(__file__)
 
+ENTRIES = [
+    {"title": "Testing Coverage", "creation_date": "Aug 22, 2016", "id": 12, "body": "I worked with Will a little bit about testing and I feel like I have a better undestanding of testing.  He clued me in on:\n\npy.test --cov=src --cov-report=term-missing\n\nThis will give you the coverage tests, but also show you which lines your code didn't hit."},
+    {"title": "Http Server Classes", "creation_date": "Aug 21, 2016", "id": 11, "body": "Today I put the response / request functions of the http server into calsses.\nI was really happy with the results and I think it was a useful exercise to do.\nMy next goal is to use the properties method in classes. \nAlso I learned today that is the server has a print line in it's implimentation it causes the whole system to hang.  It took my a long time to figure that out."},
+    {"title": "File Path Name", "creation_date": "Aug 20, 2016", "id": 10, "body": "os.path.isdir(target) - determines if the target is a directory.\nos.path.abspath(target) - gets the absolute path to the file.\nos.path.join() - joins two paths together.\nWill return an absolute path if one is an absolute path"},
+]
+
+@view_config(route_name='home', renderer='templates/index.jinja2')
 def list_view(request):
-    imported_text = open(os.path.join(HERE, 'templates/index.html')).read()
-    return Response(imported_text)
+    return {'title': 'Home',
+            "entries": ENTRIES
```

Comment:
You might want to add a test of the fact that a list of entries is in there, though.  That could be useful to know.